### ISSUE TITLE
Ability to hide/disable OSD menu on boot.

### ISSUE THREAD

**Issue by janimator0** [MiSTer-devel/Main_MiSTer#899]
I was not able to find a way to disable/hide the OSD menu on boot. This would be useful in many circumstances where settings on the MiSTer are not meant to be changed by the person holding a controller.   (In this case the menu would be accesses by a separate device)

In my case I would like my son to be able to turn on the mister on his own without the possibility of him accidentally changing complex settings on the mister when it boots. I have external ways for my son to select games that doesn't need him to open the menu.

I can see this also being useful in arcade machines where users are not meant to be able to access the menu by restarting the MiSTer device.

**Comment by janimator0** [comment]
Fair enough. Wasn't sure if this type of change would prove to be complex.

**Comment by HenrikErlandsson** [comment]
Very nice request! @janimator0  As a workaround, could set a default start core? Would that skip the OSD initially @sorgelig ?

Then, the F12/menu button could be put inside the cab to force asking manager to change the game.

**Comment by sorgelig** [comment]
Core auto-boot skips the Menu OSD, so it can be a workaround.

**Comment by JayGameHive** [comment]
Is Core auto-boot the feature where you see a loading bar before the previously played game boots? And if you interrupt the loading bar you are able to use the menu?

**Comment by HenrikErlandsson** [comment]
@JayGameHive how do you interrupt the loading? I'm not sure there is a way, short of connecting a wire to the Reset jumper on the Mister board. I tested just now with all Jamma inputs, test/service and both coins.

IDK if controllers have a Mister Menu button, but if the menu button/F12 or switch is accessible... well, then it is. Hotglue it maybe?

I think if you want to leave the Mister unattended, it best to install some sort of front-end like a game menu or make RFC game cards to swipe to change game. A mate installed that in our office lounge, which makes it snazzy and enjoyable!

At the moment, when the Menu button is pressed you land in the game's menu, with DIP switch settings etc, and have to press Right, then Fire to get back to the game list.

I guess the request could be implemented with a config setting that makes the Menu button take you right back to the game list?
Edit: Or the possibility of binding another button or control to perform an "F12 Rightarrow Enter" macro.

**Comment by birdybro** [comment]
This can be closed. There is a new setting to prevent children from mangling your settings and the user can boot into a core directly as stated. Thankfully my birds haven't figured out how to turn on the MiSTer yet.

https://github.com/MiSTer-devel/Main_MiSTer/blob/master/MiSTer.ini#L362-L371
